pkg/aws/policies.go

```diff
@@ -730,6 +734,10 @@ func (c *awsClient) DeleteOperatorRole(roleName string) error {
 		}
 		return err
 	}
+	err = c.deleteOperatorRolePolicies(policies)
```

Comment:
Operator role policies are shared across many clusters. Deleting a policy will likely fail as it will be attached to the operator roles of other clusters. _If_ it succeeds (it shouldn't) it would break those other clusters.

I think this function should check whether this role is attached to any other roles, and only delete them if no role is using it. Alternatively, this should only be called for `rosa delete account-roles`.